aws/table_aws_oam_link.go

```diff
@@ -0,0 +1,172 @@
+package aws
+
+import (
+	"context"
+
+	"github.com/turbot/steampipe-plugin-sdk/v4/grpc/proto"
+	"github.com/turbot/steampipe-plugin-sdk/v4/plugin"
+	"github.com/turbot/steampipe-plugin-sdk/v4/plugin/transform"
+
+	"github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/aws/aws-sdk-go-v2/service/oam"
+	"github.com/aws/aws-sdk-go-v2/service/oam/types"
+)
+
+func tableAwsOAMLink(_ context.Context) *plugin.Table {
+	return &plugin.Table{
+		Name:        "aws_oam_link",
+		Description: "AWS OAM Link",
+		Get: &plugin.GetConfig{
+			KeyColumns: plugin.SingleColumn("arn"),
+			IgnoreConfig: &plugin.IgnoreConfig{
+				ShouldIgnoreErrorFunc: shouldIgnoreErrors([]string{"ResourceNotFoundException"}),
```

Comment:
```suggestion
				ShouldIgnoreErrorFunc: shouldIgnoreErrors([]string{"InvalidParameterException", "ResourceNotFoundException"}),
```

Reply:
I think we missed adding `InvalidParameterException` as mentioned in https://docs.aws.amazon.com/OAM/latest/APIReference/API_ListLinks.html

Reply:
Do we need to add a retry config for the `InternalServiceFault` error as mentioned in https://docs.aws.amazon.com/OAM/latest/APIReference/API_ListLinks.html?

Reply:
I don't think we should add `InternalServiceFault` to ignore config, but we should add `InvalidParameterException` to ignore config